src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -504,6 +503,31 @@ const addKeyToCollectionItems = (list) => {
     });
 }
 
+const BOOLEAN_MAPPING = 'BOOLEAN';
+const PICKLIST_MAPPING = 'PICKLIST';
+
+const isTrueFalsePicklist = (fieldMapping) => {
+    if (fieldMapping) {
+        return fieldMapping.Target_Field_Data_Type === BOOLEAN_MAPPING &&
+            fieldMapping.Source_Field_Data_Type === PICKLIST_MAPPING;
```

Comment:
I believe new convention here to align with NGOhana is to start the new line with `&&`.

`&& fieldMapping.Source_Field_Data_Type === PICKLIST_MAPPING;`